content/lua/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: 'Data Type' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```

Comment:
```suggestion
Title: 'Data Types' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```